package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm not sure why this is here, in a C# program (since I've only seen it used with npm), but I'm fine with putting it back like it was for now.